GAE/src/org/waterforpeople/mapping/app/web/rest/QuestionRestService.java

```diff
@@ -204,8 +203,8 @@ private static void setQuestionOptionsIdList(List<QuestionDto> questionDtoList,
         Question q = questionDao.getByKey(id);
         QuestionDto dto = null;
         if (q != null) {
-            dto = new QuestionDto();
-            DtoMarshaller.copyToDto(q, dto);
+            QuestionDtoMapper qdm = new QuestionDtoMapper();
+            dto = qdm.transform(q);
             if (q.getType() == Question.Type.OPTION) {
```

Comment:
In this `if () {}` block we copy question options as well.. maybe you can replace the direct use of the spring library with our `QuestionOptionDtoMapper` as well?